sw/device/lib/dif/BUILD

```diff
@@ -0,0 +1,605 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+clkmgr_srcs = ["dif_clkmgr.c","dif_warn_unused_result.h"]
```

Comment:
You can make `dif_warn_unused_result` a cc_library and then include it as a `deps` target in the traditional way.

```
cc_library(
    name = "dif_warn_unused_result",
    hdrs = ["dif_warn_unused_result.h"],
)
```

Reply:
It looks like it was added to lib/base upstream

Reply:
I can probably simplify things by doing this for the new file dif_base.h